rockhound/registry.py

```diff
@@ -2,7 +2,6 @@
 Create a dataset registry using Pooch and the rockhound/registry.txt file.
 """
 import os
-
```

Comment:
Why are these lines deleted? I imagine it's an automatic editor thing. Could you please add them back?

Reply:
Changed